cves/CVE-2012-0858.yml

```diff
@@ -273,8 +270,8 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: this had nothing to do with internationalization it was just a invalid free that could cause a denial of service if abused
```

Comment:
You could shorten this to something along the lines of "Vulnerability did not involve 18n concerns."